tools/generators/pbxproject_prefix/src/Generator/CalculatePBXProjectPrefix.swift

```diff
@@ -0,0 +1,40 @@
+import PBXProj
+
+extension Generator {
+    /// Calculates the `PBXProject` prefix `PBXProj` partial.
+    static func calculatePBXProjectPrefix(
```

Comment:
IIUC this is really "generating" the prefix partial and not "calculating" it correct?

Reply:
In my mind generation is the whole thing, which is the combination of calculating it and then writing it. It's not generated until it's written to disk.

Reply:
Went with `calculate()` for this and `write()` for the other function. Since they are the primary functions of the generator, this kinda makes it clearer (by removing a confusing second part of the name).